### DISCUSSION TITLE
Set shortcut keys

### DISCUSSION THREAD

**Discussion by liuzhe101** [dbeaver/dbeaver#18255] [Q&A] [answered]
How should I set the shortcut keys for executing sql and opening the sql editing window

**Comment by E1izabeth** [comment, marked as answer]
You can change key binding in `Preferences->User Interface->Keys`
![image](https://user-images.githubusercontent.com/28875055/202454428-680510d7-6c08-426a-a668-1ff09fb193d4.png)

**Comment by liuzhe101** [comment]
> You can change key binding in `Preferences->User Interface->Keys` ![image](https://user-images.githubusercontent.com/28875055/202454428-680510d7-6c08-426a-a668-1ff09fb193d4.png)

thank you very much
